client/src/main/java/org/spine3/client/CommandFactory.java

```diff
@@ -37,12 +38,13 @@
 public class CommandFactory {
 
     private final UserId actor;
-
     private final ZoneOffset zoneOffset;
+    private final String boundedContextName;
 
-    protected CommandFactory(UserId actor, ZoneOffset zoneOffset) {
+    protected CommandFactory(UserId actor, ZoneOffset zoneOffset, String boundedContextName) {
```

Comment:
Again, we need to avoid this parameter. 

Even if we need info about a  `BoundedContext`, we should pass its instance and have it as a first parameter because it has more “weight” than a user — it doesn't change that often, while users and zones do.